docs/open-sdg-features.md

```diff
@@ -30,6 +43,18 @@ It is possible to "boost" one or more fields in the search index, so that more r
 
 It is also possible to index additional metadata fields, so that more fields are searched e.g. if you wanted the search to return pages where the graph title contains the search term. See the [search_index_extra_fields configuration option](https://open-sdg.readthedocs.io/en/latest/configuration/#search_index_extra_fields) for how to configure this.
 
+## Accessibility High Contrast version
+As well as the default contrast version, Open SDG also offers a high contrast version. By default two menu buttons show, to allow users to choose between the different contrast levels, for example on the [US site](https://sdg.data.gov/). Another approach is to use a contrast toggle button, for example on the [UK site](https://sustainabledevelopment-uk.github.io/).
```

Comment:
This is out of date also. There is no longer a "contrast_type" setting, and the only option is the toggle button. I do think it's worth describing here, but there doesn't need to be links to any docs.